src/mcts/params.cc

```diff
@@ -316,6 +316,9 @@ const OptionId SearchParams::kMaxCollisionVisitsScalingEndId{
 const OptionId SearchParams::kMaxCollisionVisitsScalingPowerId{
     "max-collision-visits-scaling-power", "MaxCollisionVisitsScalingPower",
     "Power to apply to the interpolation between 1 and max to make it curved."};
+const OptionId SearchParams::kEnablePendingSearcherSpinBackoffId{
+    "enable-pending-searcher-spin-backoff", "EnablePendingSearcherSpinBackoff",
```

Comment:
Maybe a shorter command line option name would be more convenient.

Reply:
The option name has been a struggle for me.
How do you think of "kSearcherSpinBackoffId", "searcher-spin-bakcoff" "SearcherSpinBackoff"